app/js/filterFunctions.js

```diff
@@ -0,0 +1,97 @@
+
+document.getElementById('filterScorePercentage').addEventListener('change', (e) => {
+    updateScoreTable()
+})
+
+/**
+ * checks the start and end date input on admin page and assigns a default value if no value set
+ *
+ * @returns array of start and end date
+ */
+function setDate() {
+    let startDateInput = document.getElementById('startDate').value
+    let startDate = '1970-01-01'
+    let endDateInput = document.getElementById('endDate').value
+    let endDate = '2099-12-31'
```

Comment:
cant you just use todays date generated dynamically through the new Date() object?